compiler/packages/babel-plugin-react-compiler/src/SSA/EnterSSA.ts

```diff
@@ -301,9 +301,6 @@ function enterSSAImpl(
         entry.preds.add(blockId);
         builder.defineFunction(loweredFunc);
         builder.enter(() => {
-          loweredFunc.context = loweredFunc.context.map(p =>
-            builder.getPlace(p),
-          );
```

Comment:
Note that we already rewrite `loweredFunction.context` in `mapInstructionOperands`